runtime/taskcontext.go

```diff
@@ -80,25 +84,38 @@ type TaskContextController struct {
 }
 
 // NewTaskContext creates a TaskContext and associated TaskContextController
-func NewTaskContext(tempLogFile string, task TaskInfo) (*TaskContext, *TaskContextController, error) {
+func NewTaskContext(tempLogFile string,
+	task TaskInfo,
+	server webhookserver.WebHookServer) (*TaskContext, *TaskContextController, error) {
 	logStream, err := stream.New(tempLogFile)
 	if err != nil {
 		return nil, nil, err
 	}
 	ctx := &TaskContext{
-		logStream: logStream,
-		TaskInfo:  task,
+		logStream:   logStream,
+		logLocation: tempLogFile,
```

Comment:
It's used later to open up a readseekcloser to the actual temp file that was created for storing the log.